src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java

```diff
@@ -1,6 +1,21 @@
-/*
- * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
- * Click nbfs://nbhost/SystemFileSystem/Templates/GUIForms/JDialog.java to edit this template
+/**
```

Comment:
This doesn't look correct as it is set as Javadoc and not a comment, have you tested this via `mvn clean verify` using the latest `license-maven-plugin` version?